### DISCUSSION TITLE
Creation of Global Variables by Organization-level Admins

### DISCUSSION THREAD

**Discussion by ben-balk-oc** [terrakube-io/terrakube#1531] [Q&A]
The Terrakube documentation (https://docs.terrakube.io/user-guide/organizations/global-variables) says: "Only users that belongs to Terrakube administrator group can create global variables. This group is defined in the terrakube settings during deployment, for more details see [Administrator group](https://docs.terrakube.io/getting-started/security#administrator-group)" 

Are there any plans to allow members of a group that has all permissions within an Organization to be able to manage Global Variables within that Organization? This would be really useful for our users since we want to allow certain admins within a team to control global variables but don't want to necessarily give them admin access to all of our Organizations.

**Comment by alfespa17** [comment]
Hello @ben-balk-oc recently we added support to manage "collection" that will be the equivalent of "variable sets" in this PR:

https://github.com/AzBuilder/terrakube/pull/1506

That feature is using a new "permission" that you can grant to specific teams and will be more flexible that using global variables.

There is an open issue also to add support to manage the configuration using the Terrkube provider here

https://github.com/AzBuilder/terraform-provider-terrakube/issues/87
